module/epochs/epoch_config.go

```diff
@@ -42,7 +42,7 @@ func DefaultEpochConfig() EpochConfig {
 // transaction argument for the deployEpoch transaction used during execution
 // state bootstrapping.
 //
-// The resulting argument has type [{String: UInt64}] which represents a list
+// The resulting argument has type [{Name: UInt64}] which represents a list
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig